build/images/ovs/Dockerfile.windows

```diff
@@ -14,34 +14,37 @@
 
 ARG NANOSERVER_VERSION
 
-FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as ovs-base
 
 ARG WIN_OVS_VERSION
 
-# Switch to a privileged user
-USER ContainerAdministrator
-
 # Sets the shell to be PowerShell 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
 
 # Install VC for ovs
 RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
-    Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait; \
-    rm C:\vc_redist.x64.exe
```

Comment:
After this change, Although ovs-base will not get used directly  , however to keep ovs-base image clean, we could still remove C:\vc_redist.x64.exe after installing C:\vc_redist.x64.exe

Reply:
The ovs-base is a temp stage for the image, and the downloaded files are not included the final image.

Reply:
In multi stage/layer docker build like this, the initial layers are temp layers, however I noticed that temp image size sometimes takes lots of space during their own layer build so while creating final image, sometimes it fails during final stage/layer build due to space issue on device. However in this case, I am fine to keep these optimisation lines to remove downloaded files in temp image or remove it .

Reply:
From Ubuntu image, I didn't find a cleanup on the downloaded files in the temp stage.

Reply:
Ubuntu image does cleanup as possible as it can: https://github.com/antrea-io/antrea/blob/aa94a3539282c19529d30a72c36f565ea66c8aae/build/images/ovs/Dockerfile#L39
I also feel no need to remove cleanup steps. They do affect the disk usage of the build machine.

Reply:
updated.